kpops/cli/main.py

```diff
@@ -233,16 +232,6 @@ def generate(
     verbose: bool = VERBOSE_OPTION,
     template: bool = typer.Option(False, help="Run Helm template"),
     steps: Optional[str] = PIPELINE_STEPS,
-    api_version: Optional[str] = typer.Option(
-        None, help="Kubernetes API version used for Capabilities.APIVersions"
-    ),
-    ca_file: Optional[Path] = typer.Option(
-        None,
-        help="Path to CA bundle file to verify certificates of HTTPS-enabled servers",
-    ),
-    cert_file: Optional[Path] = typer.Option(
-        None, help="Path to SSL certificate file to identify HTTPS client"
-    ),
```

Comment:
note to reviewers:
these are not necessary here and make the handling a little ugly, i.e. `HelmTemplateFlags` in base `PipelineComponent`)
CA bundle and Cert file are configured on component level
Kubernetes API version can now be configured in global config.yaml